### DISCUSSION TITLE
ghcr.io/wg-easy/wg-easy container can't be found in portainer

### DISCUSSION THREAD

**Discussion by slaygirlz** [wg-easy/wg-easy#739] [General]

**Comment by jenishngl** [comment]
what exactly is the issue here?

**Comment by pheiduck** [comment]
@progamer562 Please provide us with more information to help. Thank you!

**Comment by slaygirlz** [comment]
sorry I didn't get notifications but on my portainer and changing the wg old container to the new one portainer can't find the image I tried going on docker and finding it there but not showing.

**Comment by slaygirlz** [comment]
no its still saying No such image: ghcr.io/wg-easy/wg-easy:latest its even not on docker site

**Reply by centralhardware** [reply]
wg-easy use GitHub docker registry

**Comment by slaygirlz** [comment]
oh how do i add github then?

**Reply by centralhardware** [reply]
You don't need do anything. ghcr.io in image name already define registry location.

**Comment by pheiduck** [comment]
![grafik](https://github.com/wg-easy/wg-easy/assets/47042125/adbc639b-7c7d-4c2c-81ec-22b8f07cd293)

**Reply by pheiduck** [reply]
![grafik](https://github.com/wg-easy/wg-easy/assets/47042125/fff2d6cb-6494-4821-b1af-9c0bf228b9e4)

**Comment by pheiduck** [comment]
@progamer562 
Try this: https://docs.portainer.io/admin/registries/add/custom
Name: GitHub Container Registry
Registry URL: ghcr.io
Authentication: OFF / Disabled
click Add registry

Search for: **wg-easy/wg-easy** and the image will be pulled

**Comment by slaygirlz** [comment]
thank you it works but why github not docker?

**Reply by pheiduck** [reply]
Since the project is maintained by 3 people, GitHub Container Registry is preferred for authorization reasons.

**Reply by slaygirlz** [reply]
thank you tho
